Src/MoneyFox.Ui/Views/Accounts/AccountModification/AccountViewModel.cs

```diff
@@ -87,4 +87,9 @@ public override int GetHashCode()
     {
         return Id.GetHashCode();
     }
+
+    public override string ToString()
```

Comment:
if you make a dedicated vm or use the `AccountPickerViewModel` you don't need this here.

Reply:
Removed this.